allure-report-face/src/styles.css

```diff
@@ -56,6 +60,7 @@ html, body {
 }
 
 .long-line {
+  line-height: 1.5em;
```

Comment:
Not usually, it was occasional. Consider this block

```hbs
<h1 class="testcase__title long-line">{{testcase.title}}<h1>
```

I would like to have my own line-height, but I have to override it from `long-line` first.